src/components/site/SiteNavButton.md

```diff
@@ -0,0 +1,3 @@
+```jsx
+<SiteNavButton />
+```
```

Comment:
An HTML snippet could be helpful here, especially with the impending move to SE of these components.

Reply:
@theilman7 I removed the example (md file) in favor of adding a test. We shouldn't need this component in the documentation (at least I don't think) because we only have one instance of component and I dont think well need it again.